kn/fotos/api.py

```diff
@@ -42,7 +42,8 @@ def album_json(album, user):
         current_album = current_album.get_parent()
 
     return {'children': json_children,
-            'parents': json_parents}
+            'parents': json_parents,
+            'visibility': album.visibility[0]}
```

Comment:
Visibility is altijd maar één element in client-side code (en de API output dus), en meerder elementen in de server en database.